json-endpoints.php

```diff
@@ -3023,3 +3464,36 @@
 		'headers' => array( 'authorization' => 'Bearer YOUR_API_TOKEN' ),
 	),
 ) );
+
+new WPCOM_JSON_API_Publicize_Endpoint( array(
```

Comment:
Since this api endpoint is wp.com only I think this WPCOM_JSON_API_Publicize_Endpoint shouldn't be part of the this file.